airbyte-integrations/connectors/source-strava/source_strava/schemas/athlete_stats.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
+  "additionalProperties": true,
```

Comment:
Question for my own knowledge: Whats the impact to the user of adding additionalProperties?
1. Can they now sync more data?
2. Could a schema change with strava cause a users connection to fail?

Reply:
I'm not sure 100% `"additionalProperties": true` param can predict normalization fail if strava return more filed than expected.